src/store.ts

```diff
@@ -140,4 +140,44 @@ export default class Store extends AinftBase {
         throw e.response.data;
       });
   }
+
+  tryOnItem({
+    appId,
+    userId,
+    itemName,
+    chain,
+    nftContractAddress,
+    nftTokenId,
+  }: ItemTryOnParams): Promise<string> {
+    const encodedItemName = encodeURIComponent(itemName);
+    const timestamp = Date.now();
+    const body = {
+      appId,
+      chain,
+      nftContractAddress,
+      nftTokenId,
+    };
+    const data = buildData(
```

Comment:
It seems buildData() ~ return axios ... lines are repeated in many places. I think the repeated patterns can be extracted to some member methods of the base class? (For now, some todo would be enough?)

Reply:
Thanks, that sounds like a good idea. I left a todo for now.